config/locales/en.yml

```diff
@@ -28,6 +28,12 @@
 #       enabled: "ON"
 
 en:
+  banner:
```

Comment:
Amy will modify these values. There are separate entries for home and dashboard because home banner must always be present, but dashboard banner doesn't.